test/accountingTests.ts

```diff
@@ -1,5 +1,38 @@
 import { expect } from 'chai';
-import { calcTradeExposure } from "../src/Accounting"
+import { 
+  calcTradeExposure,
+  calcBorrowed,
+  calcTotalMargin,
+  calcMinimumMargin,
+  calcWithdrawable,
+  calcNotionalValue,
+  calcLeverage,
+  calcLiquidationPrice,
+  calcProfitableLiquidationPrice,
+} from "../src/Accounting"
+
+
+// Basic positions
+// The user holds a position not in liquidation
+// Deposit 200 at $100, short 10 units
+const position1 = {
+  quote: 1200, base: -10, price: 100, maxLeverage: 50
```

Comment:
Do you think it matters we don't use WAD values for these? Will just need to be careful when we move to using this with real data. The math should hold. although we may need to use BN in that case

Reply:
yeah I was thinking about this, its probably better if I use WAD